reactor-netty-core/src/main/java/reactor/netty/udp/UdpServerConfig.java

```diff
@@ -170,12 +170,16 @@ protected EventLoopGroup eventLoopGroup() {
 			AdvancedByteBufFormat.HEX_DUMP
 					.toLoggingHandler(UdpServer.class.getName(), LogLevel.DEBUG, Charset.defaultCharset());
 
-	static final class MicrometerUdpServerMetricsRecorder extends MicrometerChannelMetricsRecorder {
+	public static class MicrometerUdpServerMetricsRecorder extends MicrometerChannelMetricsRecorder {
 
 		static final MicrometerUdpServerMetricsRecorder INSTANCE = new MicrometerUdpServerMetricsRecorder();
 
-		MicrometerUdpServerMetricsRecorder() {
-			super(reactor.netty.Metrics.UDP_SERVER_PREFIX, "udp");
+		public MicrometerUdpServerMetricsRecorder(String prefix) {
+			super(prefix, "udp");
+		}
+
+		public MicrometerUdpServerMetricsRecorder() {
+			this(reactor.netty.Metrics.UDP_SERVER_PREFIX);
 		}
```

Comment:
Is there a value of making this public? One can directly use `MicrometerChannelMetricsRecorder`.

Reply:
Removed. It just helps the user configure a prefix without the need to know the exact metrics recorder they use.